bw2data/logs.py

```diff
@@ -96,7 +95,10 @@ def upload_logs_to_server(metadata={}):
     files = {"file": (uuid.uuid4().hex + ".zip", zip_fo.read())}
     metadata["json"] = "native" if anyjson is None else anyjson.implementation.name
     metadata["windows"] = config._windows
-    return requests.post(url, data=metadata, files=files)
```

Comment:
Please just delete this function instead, it is out of date for years. This URL doesn't exist.